raiden-ts/tests/unit/raiden.spec.ts

```diff
@@ -418,4 +428,74 @@ describe('Raiden', () => {
     raiden.planUDCWithdraw(withdrawBlock);
     await expect(payload).resolves.toEqual(expect.objectContaining({ block: withdrawBlock }));
   });
+
+  test('withdrawFromUDC', async () => {
+    const withdrawAmount = 10;
+
+    function udcWithdrawEpicMock(action$: Observable<RaidenAction>) {
+      return action$.pipe(
+        filter(udcWithdraw.request.is),
+        mapTo(
```

Comment:
This works fine because you **know** which parameters you're going to dispatch, but I'd say it'd be simpler and more correct to use `map` and pull some parameters (like `action.meta` and maybe some values from `payload`) from it.

Reply:
👍